lib/net/tls/impl.go

```diff
@@ -41,7 +41,7 @@ func (d *TlsDialer) dial(network, address string) (*tls.Conn, error) {
 }
 
 func newTestCertificate() (tls.Certificate, *x509.Certificate, error) {
-	key, err := ecdsa.GenerateKey(elliptic.P224(), rand.Reader)
+	key, err := ecdsa.GenerateKey(elliptic.P521(), rand.Reader)
```

Comment:
https://github.com/golang/go/issues/29779 was what tipped me off to try upping the key size.

Reply:
This was a breaking change in go 1.13 as far as i can tell.

Reply:
What was breaking with P224?

Reply:
In golang 1.11 and less TLS 1.2 was the default. With the move to 1.12+ TLS 1.3 is now the default implementation. The TLS 1.3 implementation doesn't allow the use of a key as small as P224. I didn't dig in enough to find where the cutoff for size is now though.

This was a little hard to debug since from the client side and the test all that is returned is `error: tls: internal error`.